servers/navigation/navigation_path_query_parameters_2d.cpp

```diff
@@ -102,6 +102,36 @@ real_t NavigationPathQueryParameters2D::get_simplify_epsilon() const {
 	return simplify_epsilon;
 }
 
+void NavigationPathQueryParameters2D::set_included_regions(const TypedArray<RID> &p_regions) {
+	_included_regions.clear();
+	for (const RID region_rid : p_regions) {
+		_included_regions.push_back(region_rid);
+	}
+}
+
+TypedArray<RID> NavigationPathQueryParameters2D::get_included_regions() const {
+	TypedArray<RID> r_regions;
```

Comment:
you could resize here too, but then you cannot use push_back, but need to manually assign to correct index because of how Vector operate. That would ensure only one allocation, but it's more tedious to write, so if you want to pass on that, you be the judge.